cup_detection:
  ros__parameters:
    lower_mask: [0,0,180]
    upper_mask: [65,82,255]